app-tracking-protection/vpn-internal/src/main/res/layout/activity_vpn_internal_settings.xml

```diff
@@ -149,28 +149,25 @@
                 android:theme="@style/SettingsSwitchTheme"
             />
 
-            <TextView
+            <com.duckduckgo.mobile.android.ui.view.listitem.OneLineListItem
```

Comment:
No width/height defined on any `OneLineListItem` in this xml, app crashes:
```
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
```

Reply:
The style for the element with id `apptp_bugreport` (Generate AppTP bug report) looks odd. 

![generate_bug_report](https://user-images.githubusercontent.com/7963079/191793665-7283a551-2064-4e6b-8519-96ccd21a2781.png)